downstream/titles/hub/configuring-private-hub-rh-certified/master.adoc

```diff
@@ -1,12 +1,13 @@
-= Managing Red Hat Certified and Ansible Galaxy collections in automation hub
+= Managing Ansible Content Collections and Ansible Galaxy collections in automation hub
 :numbered:
 :experimental:
 
 include::attributes/attributes.adoc[]
 
-You can sync {HubName} to use the Red Hat Certified Collections available through your {PlatformNameShort} subscription or community collections available through {Galaxy}.
+You can sync {HubName} to use the Ansible Content Collections available through your {PlatformNameShort} subscription or community collections available through {Galaxy}.
+Ansible validated collections are pre-loaded on your {PrivateHubName} and can be updated manually by downloading their packages.
```

Comment:
Validated Collections or validated collections?

I'd rework the second part to avoid passive voice. Also this raises the question for me if it is necessary to manually update collections in a private hub? Is there no option for automatic updates?